config.json

```diff
@@ -565,6 +565,15 @@
         "strings",
         "pattern matching"
       ]
+    },
+    {
```

Comment:
You might need to `rebase` and add the new metadata: https://github.com/exercism/meta/issues/16